.github/workflows/bundle-size.yml

```diff
@@ -0,0 +1,23 @@
+name: Compressed Size
+
+on:
+    pull_request:
+    workflow_dispatch:
+
+jobs:
+    build:
+        name: Bundle size check
+        runs-on: ubuntu-latest
+
+        steps:
+            - uses: actions/checkout@v2
+            - name: Use desired version of NodeJS
+              uses: actions/setup-node@v2
+              with:
+                  node-version-file: '.nvmrc'
+                  cache: npm
+            - uses: preactjs/compressed-size-action@v2
```

Comment:
@fashuvo Just wanted to note that`preactjs/compressed-size-action` is a 3rd party action and I've noticed that some of our repos uses this action. I don't prefer using 3rd party runners on our setup mostly because it can access repository secrets and if compromised, could access other private repositories via the GH token supplied. More info on this here - https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions

If we don't have an alternative method, we could go with the current approach, but we would need to watch out for security issues.